### PR TITLE
Update git command that will support old version of git also

### PR DESCRIPTION
**Pull request by aman00323**
Resolve git directory issue of https://github.com/BrainMaestro/composer-git-hooks/issues/88

### CODE REVIEW

**Comment by aman00323** [comment]
@BrainMaestro I thought test case would be run, but it is getting failure for working tree could you assist me what am i doing wrong

**Comment by BrainMaestro** [comment]
Looks like it doesn't work correctly. Does the test pass successfully for you?

**Comment by aman00323** [comment]
After changing command mkdir() in AddCommandTest() return **Not Directory** error, is there anything need to change in test cases.

**Comment by aman00323** [comment]
@BrainMaestro as i have look toward test cases, for AddCommandTest() which is finding for directory which is not their, do you show me from where i can solve this

File: src/helpers.php [hunk 1]
```diff
@@ -47,6 +47,6 @@ function global_hook_dir()
      */
     function absolute_git_dir()
     {
-        return trim(shell_exec('git rev-parse --absolute-git-dir'));
+        return trim(shell_exec('git rev-parse --git-dir'));
```
Comment (BrainMaestro) [inline src/helpers.php]: the problem is `--git-dir` resolves to `.git` if in the same directory. you need to add `realpath(trim(shell_...`

File: src/helpers.php [hunk 2]
```diff
@@ -47,6 +47,6 @@ function global_hook_dir()
      */
     function absolute_git_dir()
```
Comment (BrainMaestro) [inline src/helpers.php]: also, can you rename the function now?

Comment (aman00323) [reply src/helpers.php]: @BrainMaestro yes, please suggest name

File: src/helpers.php (hunk 1 above)
Comment (aman00323) [reply src/helpers.php]: @BrainMaestro Yea i missed that, now it is working

File: src/helpers.php (hunk 2 above)
Comment (BrainMaestro) [reply src/helpers.php]: just `git_dir` is good enough

**Comment by aman00323** [comment]
@BrainMaestro Done changes

**Comment by BrainMaestro** [comment]
Thanks! Slightly unrelated, I think the worktree test is broken somehow.

**Comment by aman00323** [comment]
@BrainMaestro i thinks test cases are working fine

**Pull request merged**